docs/environments/pacman.md

```diff
@@ -0,0 +1,66 @@
+# PacMan Environment
+
+<p align="center">
+        <img src="../env_anim/pacman.gif" width="600"/>
+</p>
+
+We provide here a minimal Jax JIT-able implementation of the game pacman. The game is played in a 2D matrix where a cell is a free space (black), a wall (dark blue), pacman (yellow) or a ghost.
+
+
+The goal is for the agent (yellow) to collect all of the pellets (small pink blocks) on the map without touching any of the ghosts. The agent receives a reward of +10 when collecting a pellet for the first time and pellets are removed from the map after being collected.
+
+The power-ups (large pink blocks) trigger a 'scatter mode' which changes the colour of the ghosts to dark blue for 30 in game steps. When the ghosts are in this state, the player can touch them which causes them to return to the center of the map. This gives a reward of +200 for each unique ghost.
+
+The agent selects an action at each timestep (up, left, right, down, no-op) which determines the direction they wil travel for that step. However, even if an action is in an invalid direction it will still be taken as input and the player will remain stationary. If the no-op action is used the player will not stop but instead take the last action that was selected.
+
+The game takes place on a fixed map and the same map is generated on each reset. The generator can be used to generate new maps based on an ASCII representation of the desired map.
+
+## Observation
+As an observation, the agent has access to the current maze configuration in the array named
+`grid`. It also has access to its current position `player_locations`, the ghosts' locations
+
+`ghost_locations`, the power-pellet locations `power_up_location`, the time left for the scatter state `frightened_state_time`, the pellet locations `pellet_locations` and the action
+mask `action_mask`.
+
+- `agent_position`: Position(row, col) (int32) each of shape `()`, agent position in the maze.
+
+- `ghost_locations`: jax array (int) of shape `(4,4)`, with the (y,x) coordinates of each ghost
+
+- `power_up_locations`: jax array (int) of shape `(4,4)`, with the (y,x) coordinates of each power-pellet
+
+- `pellet_locations`: jax array (int) of shape `(4,4)`, with the (y,x) coordinates of each pellet
+
+- `frightened_state_time`: jax array (int32) of shape `()`, number of steps left of the scatter state.
+
+- `action_mask`: jax array (bool) of shape `(4,)`, binary values denoting whether each action is
```

Comment:
The action mask is of shape (4,) but the action space is described below to be of size 5.

Reply:
You are correct. It has been updated